packages/framework/src/Model/Seo/HreflangLinksFacade.php

```diff
@@ -210,4 +213,17 @@ protected function getRelevantDomainIds(int $currentDomainId): array
 
         return [$currentDomainId, ...$alternativeDomainIds];
     }
+
+    /**
+     * @param int $domainId
+     * @param \Shopsys\FrameworkBundle\Model\Seo\Page\SeoPage $seoPage
+     * @return \Shopsys\FrameworkBundle\Model\Seo\HreflangLink
+     */
+    protected function createHreflangLinkForSeoPage(int $domainId, SeoPage $seoPage): HreflangLink
+    {
+        return new HreflangLink(
+            $this->domain->getDomainConfigById($domainId)->getLocale(),
+            $this->domain->getUrl() . $seoPage->getPageSlug($domainId),
```

Comment:
I'm not really sure where I should put it, so here it is. 
It seems like we have to somehow add slashes on the proper places.

![Snímek obrazovky 2024-11-25 v 17 49 23](https://github.com/user-attachments/assets/8a0d3e89-406f-40f3-afba-d36e7a6205d7)

![Snímek obrazovky 2024-11-25 v 17 51 44](https://github.com/user-attachments/assets/93b0318d-56f2-4225-9f62-aef48b5baf2b)

Reply:
fixed